### PR TITLE
paket in magic mode 2

### PR DESCRIPTION
**Pull request by vilinski**
Sorry, cherry-picked the commit now for #25

### CODE REVIEW

**Comment by TheAngryByrd** [comment]
Thanks!

TravisCI reporting:

```
/usr/share/dotnet/sdk/1.0.1/NuGet.targets(97,5): error : Package Microsoft.DotNet.Watcher.Tools 2.0.0 is not compatible with netcoreapp1.0 (.NETCoreApp,Version=v1.0). Package Microsoft.DotNet.Watcher.Tools 2.0.0 supports: netcoreapp2.0 (.NETCoreApp,Version=v2.0) [/home/travis/build/TheAngryByrd/MiniScaffold/Content/tests/MyLib.Tests/MyLib.Tests.fsproj]
/usr/share/dotnet/sdk/1.0.1/NuGet.targets(97,5): error : One or more packages are incompatible with .NETCoreApp,Version=v1.0. [/home/travis/build/TheAngryByrd/MiniScaffold/Content/tests/MyLib.Tests/MyLib.Tests.fsproj]
```

I guess constrain `Microsoft.DotNet.Watcher.Tools` to the 1.x version

**Comment by TheAngryByrd** [comment]
Ping.  Not sure if github send notification about review notes or not.

**Comment by vilinski** [comment]
Hi! I don't see any review notes. Only your comment about TravisCI and Watcher Tools. It seems magically fixed with my two additional commits to the branch? If not, I'll try to fix it, but at the earliest in two weeks. Sorry I'm in vacations now.

**Comment by TheAngryByrd** [comment]
I guess I just don't know how to use github's review interface, sorry.  Maybe that did it?

**Comment by vilinski** [comment]
Yes, now I see the notes

File: build.cmd [hunk 1]
```diff
@@ -15,9 +10,4 @@ IF NOT EXIST build.fsx (
   .paket\paket.exe update
   packages\build\FAKE\tools\FAKE.exe init.fsx
 )
-pushd Content\
-call build.cmd
-call build.cmd Clean
-popd
```
[the remark this answers is not in the document]
Comment (vilinski) [reply build.cmd]: strange... was unintentional, restored

**Comment by TheAngryByrd** [comment]
Closing since I merged: https://github.com/TheAngryByrd/MiniScaffold/pull/35.  It doesn't contain magic mode but i've opened a new issue for it here: https://github.com/TheAngryByrd/MiniScaffold/issues/43

**Pull request closed without merging**